README.md

```diff
@@ -188,11 +188,11 @@ license as [MongoDB](https://www.mongodb.com/community/licensing). We have
 written more about what you can and cannot do under AGPL
 [here](https://github.com/hasura/graphql-engine/wiki/License-Explained). 
 
-**Commercial licenses** that bundle the Hasura GraphQL Engine with support, and
+**Commercial licenses** that bundle the Hasura GraphQL Engine with support and
 SLAs are available on request. Please feel free to contact us at build@hasura.io
 or on our [website chat](https://hasura.io). 
 
-All **other content** (except those in [`server`](server), [`cli`](cli) and
+All **other contents** (except those in [`server`](server), [`cli`](cli) and
 [`console`](console) directories) are under [MIT License](LICENSE-community).
```

Comment:
While you are at it, can you please change this line to "are **available** under **the** [MIT License](LICENSE-community)." (edits in bold)?

Reply:
Fixed it.